docs/general/pre-sales/what-is-the-difference-between-unlimited-sites.md

```diff
@@ -0,0 +1,15 @@
+---
+id: what-is-the-difference-between-unlimited-sites
+title: Difference between Unlimited Sites and Multi-Site capable
+sidebar_label: Difference between Unlimited Sites and Multi-Site capable
+---
+
+# Difference between Unlimited Sites and Multi Site capable
```

Comment:
Use intro text instead of heading:
```suggestion
Here's the difference between *multisite-capable* and *unlimited sites* in relation to Beaver Builder packages and your Beavver Builder license.
```